frontend/scss/app/components/_table.scss

```
@@ -58,6 +58,14 @@
            align-items: center;
            justify-content: center;
            cursor: pointer;
+           // Reset native <button> styling so sortable headers match the <th>
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)